### ISSUE TITLE
Apps I have commit access to via a team do not show on My Apps dashboard

### ISSUE THREAD

**Issue by cassidyjames** [flathub/flathub#4004]
I'm a member of the @flathub/endless team on GitHub, which has commit access to the manifests for Endless apps (e.g. [Aqueducts](https://github.com/flathub/com.endlessnetwork.aqueducts/)) on GitHub. However, when logging into the My Flathub dashboard at [beta.flathub.org](http://beta.flathub.org/), I don't see the Endless apps there. I've tried hitting the refresh button, as well.

@wjt has direct commit access to the same manifests—i.e. not just via a team membership—and sees the apps on his My Flathub dashboard.

Is it possible Flathub is only looking at direct user access and not accounting for team membership?

**Comment by barthalion** [comment]
Yeah, I think this is strictly tied to direct write access at the moment. I will have a look if it can be easily fixed.
